github/issues_events.go

```diff
@@ -63,6 +63,8 @@ type IssueEvent struct {
 	//     head_ref_deleted, head_ref_restored
 	//       The pull request’s branch was deleted or restored.
 	//
+	//    review_dismissed
+	//      The review was dismissed and `DismissedReview` will be populated below.
```

Comment:
Just to be consistent, let's please add back in the one more blank line comment here, like line 65 above.